src/Validations.php

```diff
@@ -11,11 +11,14 @@ class Validations
      * @var array
      */
     const SUPPORTED_FRAMERATES = [
-        23.97,
+        24000 / 1001,
         24,
         25,
-        29.97,
+        30000 / 1001,
```

Comment:
would be good to have a const with this value, not sure if there is a good name for it tho

Reply:
We can use something like `FR_29_97`, that's how we call it on plint.

Reply:
maybe we should move all to consts